web/html/src/manager/audit/subscription-matching/subscription-matching-matcher-run-panel.tsx

```diff
@@ -144,8 +144,7 @@ class MatcherScheduleButton extends React.Component<MatcherScheduleButtonProps>
   };
 
   render() {
-    const buttonClass =
-      "btn spacewalk-btn-margin-vertical " + (!this.props.matcherRunning ? "btn-success" : "btn-default");
```

Comment:
If I'm reading this correctly, the button class used to essentially show loading state here. Is this something we want to display to the user in some other way here perhaps? I'm not familiar with this specific page.

Reply:
Okay, I see what you mean now. As a small aside, since the button class is now static, we could move it onto the element itself:

```tsx
<button type="button" className="btn space..." ... />
```